src/main/kotlin/Main.kt

```diff
@@ -1,3 +1,16 @@
-fun main(args: Array<String>) {
-    println("Hello World!")
+abstract class Menu{
+    abstract fun showPointMenu()
+    abstract fun addInput(input: Int)
+
+    fun start(){
+        while(true){
+            showPointMenu()
+            val input = readln()?.toIntOrNull()?: -1
```

Comment:
```suggestion
            val input = readln().toIntOrNull() ?: 1
```
Функция `readln()` возвращает не нуллабельное значение